index.html

```diff
@@ -3013,9 +3013,20 @@ <h3>Algorithm</h3>
                   <var>nested value</var> expands to <code>@value</code>, an
                   <a data-link-for="JsonLdErrorCode">invalid @nest value</a> error
                   has been detected and processing is aborted.</li>
-                <li>Recursively repeat steps <a href="#alg-expand-each-key-value">13</a>
+                <li>Recursively repeat steps
+                  <a href="#alg-expand-property-scoped-context">3</a>,
+                  <a href="#alg-expand-property-scoped-context2">8</a>,
+                  <a href="#alg-expand-each-key-value">13</a>,
                   and <a href="#alg-expand-resolve-nest">14</a>
-                  using <var>nested value</var> for <var>element</var>.
+                  using <var>nesting-key</var> for <var>active property</var>, and
+                  <var>nested value</var> for <var>element</var>.
+                  <div class="note">Steps <a href="#alg-expand-property-scoped-context">3</a>
+                    and <a href="#alg-expand-property-scoped-context2">8</a>
+                    may update the <var>active context</var> based on a
+                    property-scoped context associated with <var>nesting-key</var>.
+                    Updates to <var>active context</var> are restricted to the
+                    recursive operation, and do not propogate to subsequent iterations
+                    on <var>nested value</var> and <var>nesting-key</var>.</div>
```

Comment:
```suggestion
                    on <var>nested values</var> and <var>nesting-key</var>.</div>
```

Reply:
Either would be correct, but this is fine.